Memory Game/GameApp/app.js

```diff
@@ -0,0 +1,123 @@
+const cards = document.querySelectorAll('.card');
+const container = document.querySelector('.container');
+cardsArray = Array.from(cards);
+const successMessage= document.querySelector('.success--hidden');
+const scoreDiv = document.querySelector('.score');
+
+let numberOfFlippedCards = 0;
+let flippedCardArray = [];
+
+
+
+function handleClick(e) {
+    numberOfFlippedCards++;
+    const el = e.currentTarget.lastElementChild.firstElementChild;
+    const imgSrc = el.getAttribute('src');
+    const id = el.getAttribute('id');
+    if (numberOfFlippedCards === 1 && !e.currentTarget.classList.contains('flipped')) {
+        e.currentTarget.classList.add('flipped');
+        flippedCardArray.push(imgSrc);
+        flippedCardArray.push(id);
+    } else if(numberOfFlippedCards === 1 && e.currentTarget.classList.contains('flipped')) {
+        numberOfFlippedCards = 0;
+    }
+
+    if (numberOfFlippedCards === 2) {
+        if (flippedCardArray[0] === imgSrc && id !== flippedCardArray[1]) {
+            e.currentTarget.classList.add('flipped');
+            numberOfFlippedCards = 0;
+            flippedCardArray = [];
+            setTimeout(function() {
+                successMessage.style.display = 'block';
+            }, 500);
+            setTimeout(function() {
+                successMessage.style.display = 'none';
+            }, 1200);
+
+        } else if (flippedCardArray[0] !== imgSrc && id !== flippedCardArray[1] && !e.currentTarget.classList.contains('flipped')) {
+            e.currentTarget.classList.add('flipped');
+            setTimeout(closeCard, 1000, e.currentTarget);
+            setTimeout(closePrevCard, 1000, cards, `${flippedCardArray[1]}`);
+            numberOfFlippedCards = 0;
+            flippedCardArray = [];
+
+        } else if (flippedCardArray[0] === imgSrc && id === flippedCardArray[1] && e.currentTarget.classList.contains('flipped')) {
+            e.currentTarget.classList.add('flipped');
+            numberOfFlippedCards = 1;
+
+        } else if (flippedCardArray[0] !== imgSrc && id !== flippedCardArray[1] && e.currentTarget.classList.contains('flipped')) {
+            numberOfFlippedCards = 1;
+        }
+    }
+
+};
+
+cards.forEach(card => card.addEventListener('click', handleClick));
+
+function closeCard(card) {
+    card.classList.remove('flipped');
+}
+
+function closePrevCard(arr, id) {
+    arr.forEach(item => {
+        if (item.lastElementChild.firstElementChild.id === id) {
+            item.classList.remove('flipped');
```

Comment:
This line is basically the `closeCard` function. I'd call that function instead of repeating the code again. 
```suggestion
            closeCard(item);
```

If you have the same code in different places, try to remove repetition as much as possible. https://en.wikipedia.org/wiki/Don%27t_repeat_yourself